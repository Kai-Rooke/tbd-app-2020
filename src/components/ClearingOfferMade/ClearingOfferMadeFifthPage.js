import React from "react";
import TextBox from "../../components/TextBox/TextBox";
import "./ClearingOfferMade.css";
import { Container, Col } from "react-bootstrap";
import UCPButton from "../Buttons/UCPButton";


const ClearingOfferMadePageFive = (props) => {
<Container>
    <h1>Before hanging up....</h1>
    <p>Please state the following</p>
    <h3>40. Please confirm email offer will be send to</h3>
    <TextBox/>
    <h3>41. Please enter course applied for</h3>
    <TextBox/>



<Col><UCPButton></UCPButton></Col>
    <Col><UCPButton></UCPButton></Col>
 </Container>

}
export default ClearingOfferMadePageFive;