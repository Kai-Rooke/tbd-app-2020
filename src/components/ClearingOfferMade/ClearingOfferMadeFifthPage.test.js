import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import ClearingOfferMadeFifthPage from './ClearingOfferMadeFifthPage';
import { BrowserRouter as Router} from 'react-router-dom';

//cleans up 
afterEach(cleanup);

it("Renders ClearingOfferMadeFifthPage", () => {
    const { asFragment } = render(<Router><ClearingOfferMadeFifthPage /></Router>);
    expect(asFragment()).toMatchSnapshot();
});