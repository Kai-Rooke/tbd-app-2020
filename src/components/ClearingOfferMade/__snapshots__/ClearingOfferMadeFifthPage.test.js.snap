// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders ClearingOfferMadeFifthPage 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <h1>
      Before hanging up....
    </h1>
    <p>
      Please state the following
    </p>
    <h3>
      40. Please confirm email offer will be send to
    </h3>
    <div
      class="mobiletextbox"
    >
      <label />
      <input
        maxlength="30"
        minlength="0"
        placeholder=""
        size="20"
        type="text"
      />
    </div>
    <h3>
      41. Please enter course applied for
    </h3>
    <div
      class="mobiletextbox"
    >
      <label />
      <input
        maxlength="30"
        minlength="0"
        placeholder=""
        size="20"
        type="text"
      />
    </div>
    <div
      class="row"
    >
      <div
        class="col"
      >
        <br />
        <a
          href="/ClearingOfferMadeFourthPage"
        >
          <button
            class="mediumbutton"
            data-testid="ucpButton"
            href="/ClearingOfferMadeFourthPage"
            primary="True"
            type="button"
          >
            Go Back​
          </button>
        </a>
      </div>
      <div
        class="col"
      >
        <br />
        <a
          href="/emptyLink"
        >
          <button
            class="mediumbutton"
            data-testid="ucpButton"
            primary="True"
            type="button"
          >
            Confirm and Continue​
          </button>
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
